dashboard/app/controllers/course_offerings_controller.rb

```diff
@@ -35,6 +35,6 @@ def quick_assign_course_offerings
   private
 
   def course_offering_params
-    params.permit(:display_name, :is_featured, :category, :assignable, :grade_levels, :curriculum_type, :header, :marketing_initiative).to_h
+    params.permit(:display_name, :is_featured, :category, :assignable, :grade_levels, :curriculum_type, :header, :marketing_initiative, :image, :cs_topic, :school_subject, :device_compatibility).to_h
```

Comment:
I'm adding in the 'image' and 'device_compatibility' fields here too since this line change likely should have been part of the [previous PR](https://github.com/code-dot-org/code-dot-org/pull/50603) involving updates to the model and controller.

Reply:
this is a good thing to think about. I would focus on what functionality you are adding in each PR and have each PR include the minimum needed to accomplish that functionality. so, by that logic, you wouldn't need 'image' and 'device_compatibility' until the next PR. this approach can lead to some big PRs, but often it can be easier to review a slightly bigger PR that shows things working end-to-end.

it's ok if you just want to leave this here this time, or this could be a good example for us to pair on to work through how you might deal with this kind of change given that you already have another PR depending on this one.

Reply:
Good point! In that case, I'll update this to just contain what is in here end-to-end!